### PR TITLE
contact_manger

### PR DESCRIPTION
**Pull request by AliSinaYOusofi**
#### Checklist

- [x ] Added description of change
- [ x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#File-Name-guidelines)
- [x ] Added tests and example, test must pass
- [ x] Relevant documentation/comments is changed or added
- [ x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [ x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ x] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

**Review by AliSinaYOusofi — COMMENTED** [review]
#documented every function as written in contributing.md

File: contact_manager/contact_manger_linked.c [hunk 1]
```diff
@@ -0,0 +1,602 @@
+#include <stdio.h>
+#include <malloc.h>
+#include <unistd.h>
+#include <assert.h>
+
+/**
+ * @file
+ * @implementing a contact manager using linked list inc
```
Comment (Panquesito7) [inline contact_manager/contact_manger_linked.c:8]: Provide a Wikipedia link in Markdown format (if available/possible).
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂

**Review by Panquesito7 — COMMENTED** [review]
I'm not sure if this is a valid algorithm 🤔 CC: @TheAlgorithms/c_maintainer, thoughts?

**Comment by alexpantyukhin** [comment]
@Panquesito7 It looks like it demonstrates work with `linked list` probably it would be better to `C/datastructures/linked_list` . But with more clean functions without relation with contact management.

**Pull request closed without merging**